src/main/webapp/skin.xml

```diff
@@ -23,6 +23,8 @@
     xsi:schemaLocation="http://www.jasig.org/uportal/web/skin https://source.jasig.org/schemas/resource-server/skin-configuration/skin-configuration-v1.2.xsd">
 
 
+    <css>css/newsreader.css</css>
```

Comment:
If you're adding it here, I'm not sure it needs adding in the JSPs like this:  <link href="<c:url value="/css/newsreader.css"/>" rel="stylesheet" type="text/css" />